code/rounds/PreRound.cs

```diff
@@ -34,13 +34,9 @@ protected override void OnStart()
 
                 foreach (Entity entity in Entity.All)
                 {
-                    if (entity is BaseCarriable carr)
+                    if (entity is BaseCarriable carr || entity is TTTAmmo || (entity.ClassInfo.Name.StartsWith("ttt_") && entity is Prop))
```

Comment:
The startswith check should be deleted to avoid issues and whack code. Instead, using groups or tags would make more sense

Reply:
Do we group or tag entities yet?

Reply:
Added tags, I'm open to suggestions for improvements, or where the tag declaration should go.